docs-samples/V4/JS/use-luis-for-language-understanding/luis-dialogs.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Botbuilder v4 SDK - Using LUIS for language understanding
+ * 
+ * This bot demonstrates how to use LUIS with dialogs to trigger multi-turn conversation flow
```

Comment:
Elaborate on what the sample is actually doing...like, this bot will respond to requests to get the weather, turn lights on and off, and it will provide feedback if the request is not understood.